.cirrus.yml

```diff
@@ -2,27 +2,121 @@ env:
   CIRRUS_CLONE_DEPTH: "20"
   CIRRUS_SHELL: bash
 
+  ARTIFACTORY_PRIVATE_USERNAME: vault-${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-private-reader
+  ARTIFACTORY_PRIVATE_PASSWORD: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-private-reader access_token]
+  ARTIFACTORY_ACCESS_TOKEN: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-private-reader access_token]
+
+  GIT_SUB_MODULE: ""
+  GRADLE_USER_HOME: ${CIRRUS_WORKING_DIR}/.gradle
+
+linux_image_template: &LINUX_IMAGE
+  image: ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-g7-latest
+  cluster_name: ${CIRRUS_CLUSTER_NAME}
+  region: eu-central-1
+  namespace: default
+  use_in_memory_disk: true
+
+linux_2_cpu_4G_template: &LINUX_2_CPU_4G
+  eks_container:
+    <<: *LINUX_IMAGE
+    cpu: 2
+    memory: 4G
+
+linux_4_cpu_6G_java_17_template: &LINUX_4_CPU_6G_JAVA_17
+  eks_container:
+    <<: *LINUX_IMAGE
+    image: ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-g7-latest
+    cpu: 4
+    memory: 6G
+
 eks_container: &CONTAINER_DEFINITION
-  image: ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:latest
+  image: ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-g7-latest
```

Comment:
it looks like this template is no longer used and can be removed

Reply:
it should be used eventually by the promote task

Reply:
Let's see if we can simplify as we fix the pipeline in another PR